docs/application/dotnet/index.md

```diff
@@ -1,39 +1,95 @@
+  The managed code is portable. Tizen .NET applications can have large portions of their logic applied to other systems supported by the .NET Core and Xamarin.Forms.
 
-- [Third Party Libraries for Tizen .NET Application](tutorials/library/library-list.md)
+## Tizen .NET Features and Components
 
-  Introduces several third party libraries that  are used to develop .NET applications. It is classified according to the use and according to the purpose of use.
+Tizen .NET enables you to build .NET applications with Xamarin.Forms and the Tizen .NET framework. Xamarin.Forms allows you to easily create a user interface, and the TizenFX API provides numerous interfaces to much of the hardware that is found in modern TV, mobile, wearable, and IoT devices.
 
-### Visual Studio Tools for Tizen [![Download](media/ic_docs_download.png)](https://marketplace.visualstudio.com/items?itemName=tizen.VisualStudioToolsforTizen)
+**Figure: Tizen .NET architecture**
 
-For more information, see [Learn more &gt;](../vstools/index.md)
+![Tizen .NET architecture](media/cs_overview.png)
+
+Tizen .NET consists of the following main components:
+
+- **.NET Core**
+
+  .NET Core is a general-purpose development platform maintained by Microsoft and the .NET community on [GitHub](https://github.com/dotnet/core). It is cross-platform, supporting Windows&reg;, macOS, and Linux, and can be used in device, cloud, and embedded/IoT scenarios.
+
+  .NET Core is composed of the following parts:
+
+  - The [.NET runtime](https://github.com/dotnet/coreclr), which provides basic services, such as a type system, assembly loading, a garbage collector, and native interop.
+  - A set of [framework libraries](https://github.com/dotnet/corefx), which provide primitive data types, application composition types, and fundamental utilities.
+  - A [set of SDK tools](https://github.com/dotnet/cli) and [language compilers](https://github.com/dotnet/roslyn) that enable the base developer experience, available in the [.NET Core SDK](https://docs.microsoft.com/en-us/dotnet/articles/core/sdk).
+  - The dotnet application host, which is used to launch .NET Core applications. It selects and hosts the runtime, provides an assembly loading policy, and launches the application. The application host is also used to launch SDK tools similarly.
+
+- **Xamarin.Forms**
+
+  Xamarin.Forms is a cross-platform UI toolkit that allows you to efficiently create native user interface layouts that can be shared across iOS, Android&trade;, Windows Phone, and Universal Windows Platform applications.
+
+- **Tizen.Wearable.CircularUI**
+
+  [Tizen.Wearable.CircularUI](https://samsung.github.io/Tizen.CircularUI/index.html) provides Tizen wearable-specific user interfaces. It is a set of extension APIs of Xamarin.Forms. Tizen.Wearable.CircularUI APIs are supported only on Tizen wearable devices with Tizen .NET support, unlike the Xamarin.Forms which support cross platforms.
+
+- **TizenFX API**
```

Comment:
Just a suggestion : I can see this section has same information with [here](https://github.com/Samsung/tizen-docs/blob/master/docs/application/dotnet/api/overview.md#tizenfx-api). Rather than managing same information in two places, how about creating a link to `api/overview.md#tizenfx-api`?

Reply:
I think api/overview.md is okay.